### PR TITLE
Add xmas logo :santa: 

### PR DESCRIPTION
**Pull request by flomonster**
Winter is coming.

### CODE REVIEW

File: front/src/main/home.jsx [hunk 1]
```diff
@@ -9,14 +9,20 @@ import operationalStudiesImg from 'assets/pictures/home/operationalStudies.svg';
...
 export default function Home() {
   const { t } = useTranslation('home/home');
 
-  const logo = new Date().getMonth() === 5 ? proudLogo : defaultLogo;
+  let logo = defaultLogo;
+  if (new Date().getMonth() === 5) {
+    logo = proudLogo;
+  } else if (new Date().getMonth() === 11) {
+    logo = xmasLogo;
+  }
```
Comment (woshilapin) [inline front/src/main/home.jsx]: How about doing this so we don't mixup the numbers (5 is June, really?).

```suggestion
  // 0-based months
  const JUNE = 5;
  const DECEMBER = 11;
  if (new Date().getMonth() === 5) {
    logo = proudLogo;
  } else if (new Date().getMonth() === 11) {
    logo = xmasLogo;
  }
```

**Review by woshilapin — APPROVED** [review]
Just missing the "save the planet" one and we should be good! 😝

**Review by theocrsb — APPROVED** [review]

File: front/src/main/home.jsx (hunk 1 above)
Comment (SharglutDev) [reply front/src/main/home.jsx]: In the rest of the front, we would do something like this (you can move the const outside the `Home` function at l.16) : 

```suggestion
  enum MONTH_VALUES {
    JUNE: 5,
    DECEMBER: 11
  }
  
  if (new Date().getMonth() === MONTH_VALUES.JUNE) {
    logo = proudLogo;
  } else if (new Date().getMonth() === MONTH_VALUES.DECEMBER) {
    logo = xmasLogo;
  }
```

**Review by SharglutDev — APPROVED** [review]
Lgtm !

**Review by woshilapin — APPROVED** [review]

**Review by woshilapin — APPROVED** [review]

**Pull request merged**